proposals/0295-codable-synthesis-for-enums-with-associated-values.md

```diff
@@ -106,23 +108,16 @@ would encode to:
 
 ```json
 {
-  "dumpToDisk": true
+  "dumpToDisk": {}
 }
 ```
 
-With the exception of the last case, this solution is closely following the default behavior of the Rust library [serde](https://serde.rs/container-attrs.html).
```

Comment:
I think that it still would be reasonable to acknowledge serde (as a foot note) .